src/watchmaker/logger/__init__.py

```diff
@@ -141,6 +143,8 @@ def prepare_logging(log_dir, log_level):
         make_log_dir(log_dir)
         log_filename = os.sep.join((log_dir, 'watchmaker.log'))
         hdlr = logging.FileHandler(log_filename)
+        oschmod.set_mode(log_filename, 0o600)
+        oschmod.set_mode(log_dir, 0o700)
```

Comment:
I don't think we should explicitly set the mode on `log_dir` here. This is a user-provided argument. The user should manage the permissions directly on whatever directory they supply. Say for example they pass `--log-dir /var/logs` -- it would be out of place for watchmaker to set the mode there.

Or alternatively, only set the permissions if watchmaker _creates_ the directory, in `make_log_dir`.

Reply:
Was just coming to the same conclusion